api/cluster-config/v1/clusterconfig_types.go

```diff
@@ -58,11 +58,16 @@ type DiscoveryConfig struct {
 	EnableDiscovery     bool `json:"enableDiscovery"`
 	EnableAdvertisement bool `json:"enableAdvertisement"`
 
-	AutoJoin bool `json:"autojoin"`
+	AutoJoin          bool `json:"autojoin"`
+	AutoJoinUntrusted bool `json:"autojoinUntrusted"`
```

Comment:
This new fields are set as required. Did you check if we need to update the various tests that uses the `clusterconfig.policy.liqo.io` CRD?

Reply:
No, tests have not to be updated, they are not tested yet because they are not testable locally (at least it is not easy), there will be added some e2e test to check trusted/untrusted peering